### DISCUSSION TITLE
Ideal way to disable wifi-networks

### DISCUSSION THREAD

**Discussion by linusgke** [Art-of-WiFi/UniFi-API-client#130] [Q&A] [answered]
Hey there!

Which way would you recommend to disable wifi-networks in certain AP groups? I have noticed that when removing AP groups from a wifi-network within UniFi's web panel, all enabled wifi-networks on the APs within that very group briefly suspend.

Is there a way to prevent this?
I'd really appreciate any help!

Kind regards
Linus Groschke

**Comment by malle-pietje** [comment, marked as answer]
@elektr0nisch unfortunately there is no way to prevent the UniFi devices from reprovisioning when their configuration changes.

**Reply by linusgke** [reply]
Oh... what a pity. Thanks for the quick help!
